examples/tuning/cta_svm/config_yamls/1st_test_acc_params_tuning_config.yaml

```diff
@@ -0,0 +1,65 @@
+type: preprocessor
```

Comment:
The test_acc of step 2 obtains the highest score of yaml in the parameter adjustment of step 3.